gap/c3c5.gi

```diff
@@ -397,7 +397,7 @@ FindHomMethodsProjective.Subfield :=  function(ri,G)
     RECOG.SetPseudoRandomStamp(G,"Subfield");
     f := ri!.field;
     if IsPrimeField(f) then
-        return false;     # nothing to do
+        return NeverApplicable;     # nothing to do
```

Comment:
Ooops, good catch! I don't understand how I missed that one. Small consolation for me is that you also missed two other `return` statements in this function ;-)

Reply:
Well, I didn't miss them, I saw them but decided I didn't want to spare the thought to work out whether they _should actually_ also be `NeverApplicable`s. But it turns out it only took a moment's thought to realise that, yes, they should.